modules/@apostrophecms/schema/index.js

```diff
@@ -656,6 +673,9 @@ module.exports = {
                 continue;
               }
             }
+            if (isParentVisible === false) {
+              continue;
+            }
```

Comment:
@boutell Would like your opinion on that:
In this file just before line `672` we add default value to hidden fields.
If this is necessary, I suppose that we should do the same thing when a parent is hidden?
WDYT?